### DISCUSSION TITLE
How to understand that activity score is not matching well with gene expression?

### DISCUSSION THREAD

**Discussion by Dongxu-Zheng** [aertslab/SCENIC#286] [* General SCENIC questions]
Hi, thanks for developing such great software!

I have a question about how to understand the result of SCENIC. The first step of the workflow is to do the co-expression analysis between TF and genes. However, when I return back to see the correlation between the TF and its target gene, I didn't find a good correlation there. In addition, the activity score of regulon is not well matching with the gene expression. So how to understand this result? 

I find there is an issue related to my question:
https://github.com/aertslab/SCENIC/issues/37

Thanks for any suggestion or response.

Best,
Dongxu

**Comment by Jiayi-Zheng** [comment]
I have the same question. I have a control and treatment dataset, where a few transcriptional factor showed similar level of expression. However, SCENIC tells me they have quite some difference in the activity score... Wonder in biological terms, how does sequencing depth and other things affect the score.
